src/javascript/JContent/EditFrame/Create.jsx

```diff
@@ -187,22 +189,37 @@ export const Create = React.memo(({element, node, nodes, addIntervalCallback, cl
         };
     };
 
+    const createButtonRef = useRef(null);
+
+    useEffect(() => {
```

Comment:
I would prefer to combine this with the useEffect in line 128 and keep all the sizing logic together